variables.tf

```diff
@@ -174,15 +174,13 @@ variable "gateway_probe_threshold" {
 
 # TLS
 variable "tls_certificate_path" {
-  description = "Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB."
+  description = "Path to a TLS certificate that will be imported in Azure Key Vault and used in the Application Gateway TLS listener for GraphDB. Either tls_certificate_path or tls_certificate_password must be provided."
   type        = string
-  default     = null
```

Comment:
return the null values

Reply:
If I return the Null the table sets them as Required if I'm not mistaken?
@mihailradkov

Reply:
> Maybe return the null values, not sure @mihailradkov WDYT?

Until we migrate to the latest Terraform that provides more complex validations including values from other variables, we need these to have `null` as a default. Especially `tls_certificate_password` because a certificate might be unprotected (bad idea but it's allowed).

> If I return the Null the table sets them as Required if I'm not mistaken? @mihailradkov

Just test it and you will figure it out.

Reply:
This is still a valid note.

Reply:
I have changed the default to NULL and now the required is set to NO. 
If no other changes are needed, I will resolve this note.